### PR TITLE
Update chat messages styling

### LINKED ISSUE #4414: Chat message style update
*Type*: Bug/Feature
- In group and 1-1 chats, the messages, margins and padding around them, typography is not implemented correctly
- Timestamps and Names should be updated
- Messages "from myself" should have a different background color
- Link color and formatting

Zeplin:
Web: https://zpl.io/Vxkpl7k 
App: zpl://screen?sid=5b02bf07d181f8871a501c30&pid=5a55d587ba304eed4d2c942f 

messages

### PR DESCRIPTION
**Pull request by dmitryn**
fixes #4414

### Summary:

* Change background color of outgoing messages
* Add more space between public chat messages

### Review notes:
Except changes mentioned above i haven't found any mismatches with Zeplin's "The Message" mockup. @andmironov please let me know if i missed something.

### Testing notes:
PR contains UI styling changes only. No logic has been changed.

status: ready

### CODE REVIEW

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/Qcbg9A for branch PR-4461

<blockquote><div><strong><a href="https://i.diawi.com/Qcbg9A">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/S8iSnp for branch PR-4461

<blockquote><div><strong><a href="https://i.diawi.com/S8iSnp">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

File: src/status_im/chat/styles/message/message.cljs [hunk 1]
```diff
@@ -48,18 +48,18 @@
...
          (when justify-timestamp? {:position :absolute
                                    :bottom   10
                                    :right    12})))
 
-(def message-timestamp-placeholder
-  (assoc (message-timestamp false)
-         :color styles/color-white))
+(defn message-timestamp-placeholder [outgoing]
+  (assoc (message-timestamp false outgoing)
+         :color (if outgoing colors/hawkes-blue styles/color-white)))
```
Comment (jeluard) [inline src/status_im/chat/styles/message/message.cljs]: Maybe there could be a more abstract fn used to compose message style? `color` is overridden here so passing `outgoing` to `message-timestamp` is void.

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/XWVyed for branch PR-4461

<blockquote><div><strong><a href="https://i.diawi.com/XWVyed">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/WLF7kp for branch PR-4461

<blockquote><div><strong><a href="https://i.diawi.com/WLF7kp">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

File: src/status_im/chat/styles/message/message.cljs (hunk 1 above)
Comment (dmitryn) [reply src/status_im/chat/styles/message/message.cljs]: @jeluard makes sense, refactored that.

**Review by jeluard — APPROVED** [review]

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/S8XYdT for branch PR-4461

<blockquote><div><strong><a href="https://i.diawi.com/S8XYdT">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/6BTQnM for branch PR-4461

<blockquote><div><strong><a href="https://i.diawi.com/6BTQnM">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/5utpRe for branch PR-4461

<blockquote><div><strong><a href="https://i.diawi.com/5utpRe">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Review by flexsurfer — APPROVED** [review]

**Comment by asemiankevich** [comment]
looks good to me

**Pull request merged**